_docs/cardano/2017-01-10-transactions.md

```diff
@@ -123,3 +123,33 @@ called "segregated witness" (you may have heard of it being recently
 [implemented in Bitcoin](https://bitcoincore.org/en/2016/01/26/segwit-benefits/)).
 Under this scheme, transactions and proofs are stored in two separate places in a block,
 and can be processed independently.
+
+## Transaction Distribution 
+
+As a part of address design, we introduced transaction distribution feature.
+
+Suppose we have an address `A`. Which stakeholders should benefit from money stored on this address?
+For [`PublicKey`](/cardano/addresses/)-address it's obvious and straightforward, though it's not for
+[`ScriptAddress`](/cardano/addresses/) (e.g. for `2-of-3` multisig address implemented via script we
+might want to have distribution `[(A, 1/3), (B, 1/3), (C, 1/3)]`).
+For any new address' type might be introduced via softfork in the future it might be useful as well
```

Comment:
>"For any new address' type might be introduced via softfork in the future it might be useful..."
I don't get it, what do you mean here? Is it:
"For any new address type introduced via softfork in the future it might be useful..."
?